src/components/Icon/Icon.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import iconSprite from 'bootstrap-italia/dist/svg/sprite.svg'
+
+const propTypes = {
+    color: PropTypes.string,
+    size: PropTypes.string,
+    icon: PropTypes.string
+}
+
+const defaultProps = {
+    color: '',
+    size: '',
+    icon: 'it-tool'
+}
+
+class Icon extends React.Component {
+    render () {
+        const {
+            color,
+            size
+        } = this.props
+        const classes = classNames('icon', color, size)
+        const iconName = this.props.icon
+        return (
+            <svg className={classes}>
+                <use xlinkHref={`${iconSprite}#${iconName}`}/>
+            </svg>
+        )
+    }
+}
+
+Icon.propTypes = propTypes
+Icon.defaultProps = defaultProps
+
+export default Icon
```

Comment:
This is something new: I centralized SVG inclusion, so that we have a convenient component that writes any `svg` code we need. Following this change, we'll need to update any component using:
```
<svg className={classes}>
                <use xlinkHref=.../>
</svg>
```

to

```
<Icon ... />
```

I did that for the Button component only, for now (as this was already getting too big).

Reply:
No worries!